src/qt/CMakeLists.txt

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+# See:
+#  - https://cmake.org/cmake/help/latest/manual/cmake-qt.7.html
+#  - https://doc.qt.io/qt-5/cmake-manual.html
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOUIC_SEARCH_PATHS forms)
+
+set(qt_minimum_required_version 5.11.3)
+
+set(qt_components Core Gui Widgets Network LinguistTools)
+
+if(CMAKE_CROSSCOMPILING)
+  # The find_package(Qt ...) function internally uses find_library()
+  # calls for all dependencies to ensure their availability.
+  # In turn, the find_library() inspects the well-known locations
+  # on the file system; therefore, it must be able to find
+  # platform-specific system libraries, for example:
+  # /usr/x86_64-w64-mingw32/lib/libm.a or /usr/arm-linux-gnueabihf/lib/libm.a.
+  set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY BOTH)
+endif()
+
+find_package(Qt5 ${qt_minimum_required_version} REQUIRED
+  COMPONENTS ${qt_components}
+  HINTS ${qt5_brew_prefix}
+  PATH_SUFFIXES Qt5 # Required on OpenBSD systems.
+)
+unset(qt_components)
+message(STATUS "Found Qt: ${Qt5_DIR} (found suitable version \"${Qt5_VERSION}\", minimum required is \"${qt_minimum_required_version}\")")
+unset(qt_minimum_required_version)
+
+file(GLOB ts_files RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} locale/*.ts)
```

Comment:
You are right. I would consider this approach as temporary one until the https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/main/update-translations.py script is updated. Specifically, its `update_build_systems` function.

Also the `bitcoin_locale.qrc` file could be a build artifact rather than a source tree file.

Reply:
Maybe add a comment to make this clear?

Reply:
> Maybe add a comment to make this clear?

Added.